x/superfluid/keeper/stake.go

```diff
@@ -48,13 +48,14 @@ func (k Keeper) RefreshIntermediaryDelegationAmounts(ctx sdk.Context) {
 			continue
 		}
 
+		currentAmount := sdk.NewInt(0)
 		delegation, found := k.sk.GetDelegation(ctx, mAddr, valAddress)
 		if !found {
-			k.Logger(ctx).Error(fmt.Sprintf("delegation not found for %s with %s", mAddr.String(), acc.ValAddr))
-			continue
```

Comment:
Why are we continuing if there is no delegation from the mAddr?

Reply:
ah thats current delegation, good catch

Reply:
I guess we do have this issue when someone fully unbonds from a validator, this will error / be at 0 each epoch.